src/bin/repl.js

```diff
@@ -3,7 +3,7 @@ import { start } from 'repl'
 import App from '../app'
 
 const app = new App(process.env.COZY_DESKTOP_DIR)
-const config = app.config.getDevice()
+const config = app.config.config
```

Comment:
Well, there were no multi-config support. I had tried it, but it was too error-prone and even if the config file can have several "devices", only the first one was really used.